mobile_app/README.md

```diff
@@ -0,0 +1,30 @@
+Mobile Apps
+===========
+
+Tools contained here are used for working with the mobile-apps
+
+trigger_build
+-------------
+This will:
+
+* Write a known set of environment variables to disk in JSON format to a file
+  named CONFIGURATION.
```

Comment:
Oh hrrrm. So I had been thinking of this in the context of the particular ``edx-build-<platform>`` repos we just made, which are essentially placeholders for kicking off a build and contain almost nothing. But I suppose there's no reason this couldn't be used more generally. I'm inclined to focus on that particular use case right now, but it would be very straightforward to generalize.